torchok/data/datasets/classification/classification.py

```diff
@@ -50,7 +77,7 @@ def __init__(self,
 
         Args:
             data_folder: Directory with all the images.
-            csv_path: Path to the csv file with path to images and annotations.
+            annotation_path: Path to the .pkl or .csv with path to images and annotations. M
```

Comment:
remove `M`. Do you understand what this name changing force you to change each config that we have?

Reply:
Removed